packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -171,6 +171,12 @@ const conf = (module.exports = convict({
         env: 'FEATURE_FLAGS_REDIS_HOST',
         format: String,
       },
+      password: {
+        default: 'fxa123',
+        doc: 'Redis password',
+        env: 'REDIS_PASSWORD',
+        format: String,
```

Comment:
Not sure if we want to add `sensitive: true` to avoid potential password leakage.
Quoth the [convict docs](https://github.com/mozilla/node-convict/tree/master/packages/convict):

> **Sensitive values and secrets:** If `sensitive` is set to `true`, this value will be masked to `"[Sensitive]"` when `config.toString()` is called. This helps avoid disclosing secret keys when printing configuration at application start for debugging purposes.

Quick grep shows this is currently only used one(?!) place in our configs:

```sh
git grep -n -B2 -A2 "sensitive:" packages | cat
```

https://github.com/mozilla/fxa/blob/7fdfe181638435144ee17f1389d27cf90b1d4c6b/packages/fxa-profile-server/lib/config.js#L181-L185

Reply:
> this value will be masked to "[Sensitive]" when config.toString() is called
SRE literally need to connect a debugger to a running service with sensitive values and print the convict object as a string?  Seems somewhat unlikely.